frontend/src/metabase/pulse/components/PulseEdit.jsx

```diff
@@ -80,13 +81,17 @@ export default class PulseEdit extends Component {
     }
 
     getConfirmItems() {
+        const recipientslen = <strong>{c.recipients.length} {inflect("address", c.recipients.length)}</strong>
```

Comment:
You can use `jt` for these to prevent having to create variables. 
```jsx
<span>{ t`This pusle will no longer be emailed to${<strong>{c.recipients.length}</strong>} ... </span>
```